stories/Molecules/FooterNavigation/FooterIcons/footerIcons.scss

```diff
@@ -48,28 +38,15 @@
     }
   }
 
-  &.white,
-  &.blue {
+  &.inverted {
     li {
       a {
-        &.facebook {
-          background: url(#{$img-path-Icon}/facebook-white.svg) no-repeat center bottom;
-        }
-
-        &.instragram {
-          background: url(#{$img-path-Icon}/instagram-white.svg) no-repeat center bottom;
-        }
-
-        &.linkedin {
-          background: url(#{$img-path-Icon}/linkedin-white.svg) no-repeat center bottom;
-        }
-
-        &.twitter {
-          background: url(#{$img-path-Icon}/twitter-white.svg) no-repeat center bottom;
-        }
+        $socials: ('facebook', 'instagram', 'undp-linkedin', 'twitter', 'youtube');
```

Comment:
@Shubham-Sharma-77 Any specific reason why this needs to be `undp-linkedin`? Otherwise change this to just `linkedin`.